### ISSUE TITLE
How to increase the font Size as inline style object is not working

### ISSUE THREAD

**Issue by highskillzz** [TeamWertarbyte/material-ui-search-bar#38]
I am unable to increase the font size through the inline object, if i try to do the seach text is going below but the text size is not working

import React, { Component } from "react";
import { connect } from "react-redux";
import { withRouter } from "react-router-dom";
import { SearchTerm } from "../actions/index";
import Button from "material-ui/Button";
import SearchBar from "material-ui-search-bar";

class Searchbar extends Component {
  constructor(props) {
    super(props);
    this.state = {
      terms: ""
    };
    this.handleSubmit = this.handleSubmit.bind(this);
    this.handleLucky = this.handleLucky.bind(this);
  }
  handleSubmit(e) {
    // e.preventDefault();
    this.props.SearchTerm(this.state.terms);
    this.setState({ terms: "" });
    this.props.history.push("/projects");
  }
  handleLucky(e) {
    e.preventDefault();
    // After implementing machine learning show the most probable favourite projects
  }
  render() {
    const style = {
      marginTop: "4vh",
      width: "35vh"
    };
    return (
      <div>
        <form onSubmit={this.handleSubmit}>
          {/*<input
            name="terms"
            value={this.state.terms}
            type="text"
            onChange={e => {
              this.setState({ [e.target.name]: e.target.value });
            }}
            style={{width:"90vh",height:"5vh",padding:"1vh",fontSize:"3vh",marginLeft:"2vh"}}
          />*/}
          <SearchBar
            onChange={e => {
              this.setState({ terms: e });
            }}
            onRequestSearch={e => this.handleSubmit(e)}
            name="terms"
            value={this.state.terms}
            
            style={{
              margin: "0 auto",
              maxWidth: 900,
              height: "8vh"
            }}
            
          />
          <br />
          <Button
            variant="raised"
            label="Search"
            type="submit"
            color="primary"
            size="large"
            style={{ ...style, marginLeft: "8vh" }}
          >
            Search
          </Button>
          <Button
            variant="raised"
            label="Feeling Lucky?"
            onClick={this.handleLucky}
            type="submit"
            color="secondary"
            size="large"
            style={{ ...style, marginLeft: "4vh" }}
          >
            Feeling Lucky?
          </Button>
        </form>
      </div>
    );
  }
}

function mapStateToProps(state) {
  console.log(state.term);
  return {
    interterms: state.term
  };
}

const MapDispatchToProps = dispatch => ({
  SearchTerm: term => {
    dispatch(SearchTerm(term));
  }
});

export default withRouter(
  connect(mapStateToProps, MapDispatchToProps)(Searchbar)
);

This is my code. I have tried putting it as a class name as well but to no use

**Comment by leMaik** [comment]
I don't see where you are trying to increase the font size? You are just resizing the search bar to 8vh (that's 8 percent of the viewport height and really not what you want :wink:). Try setting `fontSize: 8px` for a smaller font size.

**Comment by highskillzz** [comment]
@leMaik I thought that your answer would solve the problem but this is happening if i set the font size to 70px, I set it 70 to see the effect clearly but it doesnt increase the font size instead acts similar to margintop. Thanks btw
![image](https://user-images.githubusercontent.com/20688709/39408816-d9362046-4bf9-11e8-9667-40c9737431f3.png)

**Comment by highskillzz** [comment]
@leMaik I am using material-ui 1.0.0

**Comment by leMaik** [comment]
@highskillzz This component, unlike most of our our components, doesn't use `withStyles` yet (ping @saschb2b). Thus, you'll have to "hack" it by using `inputProps={{ className: <put your class to change the font size here }}` for now. We'll fix this in the next beta. You can't change `style` with `inputProps`, though, as we overwrite that.

**Comment by highskillzz** [comment]
@leMaik is this how you meant?
<SearchBar
            onChange={e => {
              this.setState({ terms: e });
            }}
            onRequestSearch={e => this.handleSubmit(e)}
            name="terms"
            value={this.state.terms}
            inputProps={{ fontSize: "80px" }}
            style={{
              margin: "0 auto",
              maxWidth: 900,
              height: "8vh",
              
            }}
            
          />

if it is, then this is the result. Am I doing it wrong somewhere?
![image](https://user-images.githubusercontent.com/20688709/39409344-18963be6-4c03-11e8-95e2-3694738e4d2d.png)

**Comment by leMaik** [comment]
@highskillzz No. Set a classname that sets the font size, e.g. define a class in your styles object and then use it with `inputProps={{ className: classes.yourClass }}`.

**Comment by highskillzz** [comment]
export default withRouter(
  connect(mapStateToProps, MapDispatchToProps)withStyles(styles)(Searchbar)
);
this doesnt work

**Comment by highskillzz** [comment]
but @leMaik it made a side effect 
![image](https://user-images.githubusercontent.com/20688709/39437812-a2cdc92a-4cbf-11e8-9e25-9ef3dca2bbab.png)
and I am unable to solve it even if I input the width into the inputProps like this

const styles = {
  
    margin: "0 auto",
    maxWidth: 900,
    height: "8vh",
    fontsSize:{fontSize:"20px",width:"75%"}
  
};

<SearchBar
            onChange={e => {
              this.setState({ terms: e });
            }}
            onRequestSearch={e => this.handleSubmit(e)}
            name="terms"
            value={this.state.terms}
            inputProps={{ className: this.props.classes.fontsSize }}
            
          />

**Comment by leMaik** [comment]
Glad you solved this.
